tests/test_create_estimator.py

```diff
@@ -0,0 +1,20 @@
+import pytest
+from yacs.config import CfgNode as CN
+
+from datasetinsights.estimators import FasterRCNN, create_estimator
+
+
+@pytest.fixture
+def config():
+    """prepare config."""
+    with open("tests/configs/faster_rcnn_groceries_real_test.yaml") as f:
+        cfg = CN.load_cfg(f)
+
+    return cfg
+
+
+def test_create_estimator_with_config_expect_faster_rcnn(config):
```

Comment:
Test against a specific estimator create will become tedious,  as developers have to remember add a new test for each new estimator that was introduced. It's very likely that developers will forgot. Test whether `FasterRCNN` estimator can be successfully initialized should be the responsibility of tests specific to each estimator, such as `test_faster_rcnn.py` or `test_deeplab.py`